<ui version="4.0" >
 <class>DebuggerWidget</class>
 <widget class="QWidget" name="DebuggerWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>653</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Mupen64plus debugger</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Disassembler</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>0</number>
        </property>
        <item row="0" column="0" >
         <layout class="QHBoxLayout" name="horizontalLayoutDisasm" >
          <item>
           <widget class="QTreeWidget" name="treeDisasm" >
            <column>
             <property name="text" >
              <string>1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title" >
        <string>Debug</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayoutButtons" >
          <property name="sizeConstraint" >
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <property name="fieldGrowthPolicy" >
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <property name="formAlignment" >
           <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
          </property>
          <item row="1" column="1" >
           <widget class="QPushButton" name="pushRun" >
            <property name="text" >
             <string>&amp;Run</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QPushButton" name="pushTrace" >
            <property name="text" >
             <string>&amp;Trace</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" >
           <widget class="QPushButton" name="pushStep" >
            <property name="text" >
             <string>&amp;Step</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1" >
           <widget class="QPushButton" name="pushBreak" >
            <property name="text" >
             <string>&amp;Break</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" >
           <widget class="QPushButton" name="pushGoto" >
            <property name="text" >
             <string>&amp;Go To...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
